src/index.js

```diff
@@ -1,8 +1,19 @@
 import './styles.scss';
 
-const listContainer = document.querySelector('.container-list');
+import Task from './modules/task.js';
+import { inputTask, addTask } from './modules/addtask.js';
+import { checkBoxStatus, changeStyleTask } from './modules/checkStatus.js';
+import {
+  dragStart, dragEnd, dragOver, dragDrop,
+} from './modules/dragDrop.js';
+import { listContainer, createItemTask } from './modules/renderTasks.js';
+import {
+  clearCompletedTasks, removeAllIcon, removeAllItems, removeCompletedItem, removeSelectedItem,
+} from './modules/remove.js';
 
-const toDoTasks = [
+// const listContainer = document.querySelector('.container-list');
```

Comment:
- [ ] Please kindly delete the commented line of code to have a clean and professional  codebase 😎 😉